test/models/network.js

```diff
@@ -46,6 +48,15 @@ describe("Network", function() {
 			});
 		});
 
+		it("should generate uuid (v4) for each network", function() {
```

Comment:
Could maybe add a simple test that checks whether we log to text files or not. Don't need too much, but it might be a nice wee test.

Reply:
@xPaw, what do you think about this?

Reply:
I'm not sure why @McInkay said text file logging, which I didn't touch in this PR?

I'll try to add some sqlite tests.

Reply:
You didn't change the text logging, but you changed the logic around when we do that. You've added the option in the config to define when we do what logging.

Reply:
The trouble with testing these code paths is that they want plenty of object references (client, network, message, config helper)

Reply:
Looks like we need to improve the code then.